io.github.Abstract_AA.Framestop.desktop

```
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Name=Framestop
+Comment=Freeze frame tool that auto selects the best frame
+Exec=/app/bin/framestop
```

this is never a full path.
```suggestion
Exec=framestop
```